content/monitors/monitor_types/anomaly.md

```diff
@@ -178,15 +178,21 @@ Not interpolating between counts makes sense if the thing you are counting is so
 
 ### How does setting the rollup interval in "Advanced Options" differ from setting it on the query using `.rollup()`?
 
-If the rollup is set explicitly on the query, the rollup interval option for the anomaly monitor will be ignored.
+If the rollup is set explicitly on the query, the rollup interval option for the anomaly monitor is ignored.
 
 ### I don't care if my metric is anomalous if its value is less than X, can I somehow ignore those anomalies?
 
 Create **A**: an anomaly monitor to alert on values above the bounds; and **B**: a separate [metric monitor][9] with a threshold alert to alert on values greater than X; and then finally a [composite monitor][10] on **A && B**.
 
 ### Why am I prevented from saving a monitor with a message like ''alert and alert recovery criteria are such that the monitor can be simultaneously in alert and alert recovery states?"
 
-Setting different windows for the alert and alert recovery periods might lead to an ambiguous state. The alert and alert recovery window sizes should be set such that both cannot be satisfied at the same time. For example, setting an alert threshold at 50% for a 2-hour window (i.e., 1 hour has to be anomalous to trigger the alert) and the recovery threshold at 50% for a 10-minute window (i.e., 5 minutes have to be non-anomalous to recover) might result in triggering the alert and the alert recovery states simultaneously. If the last 5 minutes are not anomalous but the 1 hour before that _was_ anomalous, both the alert and the alert recovery will be triggered.
+Setting different windows for the alert and alert recovery periods might lead to an ambiguous state. The alert and alert recovery window sizes should be set such that both cannot be satisfied at the same time. For example, setting an alert threshold at 50% for a 2-hour window (i.e., 1 hour has to be anomalous to trigger the alert) and the recovery threshold at 50% for a 10-minute window (i.e., 5 minutes have to be non-anomalous to recover) might result in triggering the alert and the alert recovery states simultaneously. If the last 5 minutes are not anomalous but the 1 hour before that _was_ anomalous, both the alert and the alert recovery are triggered.
+
+### How does daylight savings affect anomaly detection monitors?  
+
+Datadog monitors use UTC time and by default are agnostic to local time zones (e.g. EST, PST, CST). As a result of the daylight savings time shift, people’s behavior naturally changes as compared to the UTC time. This could be detected as unexpected anomaly. 
```

Comment:
phrasing for "people’s behavior naturally changes as compared to the UTC time" could be changed to be more clear. Eg. User activity is shifted relative to UTC time because activity typically remains the same for the user's local time.